src/__tests__/Modal.spec.js

```diff
@@ -76,7 +76,7 @@ describe('Modal', () => {
     );
 
     jest.runTimersToTime(300);
-    expect(wrapper.childAt(0).children().length).toBe(0);
+    expect(wrapper.childAt(0).children().length).not.toBe(0);
```

Comment:
You can remove all of the checks like this. The previous test was ensuring nothing was inserted into the DOM, but now using react-portal, we will always be inserting react-portal, so these checks are more/less pointless (unless you want to check that react-portal get inserted, but I would also only do that in one test, not all of them.)

Reply:
Removed those unneeded checks and created new test for this.